src/data/customer/__test__/job.test.js

```diff
@@ -1,4 +1,9 @@
-import { getJobDetailAsync, getJobSummaryAsync, createJobsAsync } from '../Job';
```

Comment:
We need to test removing a job ID that's part of a different route:

1. The route is owned by customer but it's a different one than the job ID
2. The route is not owned by customer